tests/build.gradle

```diff
@@ -250,8 +255,33 @@ gradle.projectsEvaluated {
             showStandardStreams = true
             exceptionFormat = 'full'
         }
+        maxHeapSize = "1500m"
         outputs.upToDateWhen { false } // force tests to run every time
     }
+    /**
+     * Task to generate coverage xml report. Requires the
+     * tests to be executed prior to its invocation
+     */
+    task reportCoverage(type: ScoverageReport) {
```

Comment:
The new version now creates task like `report<Test Task Name>Scoverage` as entry task. This task in turn adapts the compile task to use the instrumented classes. This works fine for conventional cases where tests are part of same module. 

However in our case as we have a separate test module and further to capture coverage of code running within container (as part of integration test) we copy the coverage metrics files manually we cannot use default tasks. Hence the report task duplicate bit of logic from actual task